core/semver/semver.go

```diff
@@ -327,3 +343,27 @@ func fileExists(filePath string) bool {
 	}
 	return false
 }
+
+// ReadFile is a wrapper around os.ReadFile
+var ReadFile = func(file string) ([]byte, error) {
```

Comment:
why do we need a wrapper around one liners? Just curious.

Reply:
Ah, thanks

Reply:
For now I added them to the same package where they are used. It would be on a per-package basis unless we wanted some common functions to be called from anywhere in the driver/reverse-proxy.